utils/remote_control/navigator_joystick_control/nodes/navigator_joystick.py

```diff
@@ -135,9 +139,7 @@ def joy_cb(self, joy):
         # Station hold
         if station_hold == 1 and station_hold != self.last_station_hold_state:
```

Comment:
[nit] Consider using an Enum (or even a string/dict) for the station_hold states, instead of naked integers